java/src/jmri/jmrix/srcp/SRCPThrottle.java

```diff
@@ -76,7 +77,7 @@ public SRCPThrottle(SRCPBusConnectionMemo memo, DccLocoAddress address,
                 + (address.getNumber())
                 + " " + protocol + " "
                 + (address.isLongAddress() ? " 2 " : " 1 ")
-                + maxsteps + " "
+                + speedStepMode.numSteps + " "
```

Comment:
This is also building a protocol string.

Reply:
Fixed.